app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :user
+
+  def update_posts_counter
+    user.update(posts_counter: user.posts.count)
+  end
```

Comment:
[Hint] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `update_posts_counter` method every time a new post has been saved to the database, then make the `update_posts_counter` method private